blocks/content-api-source-block/CHANGELOG.md

```diff
@@ -1,86 +0,0 @@
-# Change Log
```

Comment:
was this intentionally deleted? I'm not sure that we'd need to blow this away

Reply:
Assuming it's regenerated each build.  The v2 blocks haven't been generating with it, so I've been removing it but I can add it back in.